server/services/tests/messagesSerivices.test.js

```diff
@@ -0,0 +1,81 @@
+let messagesServicesFactory = require('../messagesServices');
+const chai = require("chai");
+const expect = chai.expect;
+const sinon = require("sinon");
+
+describe("messagesServices", function() {
```

Comment:
you have ignored test cases in the function sendMessageToGroup including undefined message and no users returned